### PR TITLE
Add sequelize.truncate function

### PR DESCRIPTION
**Pull request by BridgeAR**
You originally closed this since the author did not respond anymore.

### CODE REVIEW

File: test/integration/sequelize.test.js [hunk 1]
```diff
@@ -647,8 +647,27 @@ describe(Support.getTestDialectTeaser('Sequelize'), function() {
     });
   });
 
+  describe('truncate', function() {
+    it("truncates all models", function() {
+      var Project = this.sequelize.define('project' + config.rand(), { title: DataTypes.STRING });
+
+      return this.sequelize.sync({ force: true }).then(function() {
+        return Project.create({ title: 'bla' });
```
Comment (mickhansen) [inline test/integration/sequelize.test.js]: Might want to test that auto increment ids have been reset aswell? Although i'm not sure if that's applicable to all dialects.

File: lib/sequelize.js [hunk 2]
```diff
@@ -902,6 +902,31 @@ module.exports = (function() {
...
+    var models = [];
+
+    this.modelManager.forEachModel(function(model) {
+      if (model) {
+        models.push(model);
+      }
+    }, { reverse: false });
+
+    return Promise.map(models, function(model) {
+      return model.destroy(options);
```
Comment (mickhansen) [inline lib/sequelize.js]: Perhaps provide a `truncate` option on the model aswell?

Comment (BridgeAR) [inline lib/sequelize.js]: Done

File: test/integration/sequelize.test.js (hunk 1 above)
Comment (BridgeAR) [inline test/integration/sequelize.test.js]: Puh... I thought about something like that while writing this but I hoped you didn't mind me being lazy :D

File: lib/sequelize.js (hunk 2 above)
Comment (mickhansen) [inline lib/sequelize.js]: And then use it here :p

**Comment by BridgeAR** [comment]
Something that you'd like to be further improved before merging?

**Comment by janmeier** [comment]
Nope, this look good :)

**Pull request merged**